app/controllers/api/v1/reviews_controller.rb

```diff
@@ -0,0 +1,29 @@
+class Api::V1::PostsController < ApplicationController
+  def index
+    render json: {Review.all}
+  end
+
+  def show
+    render json: {Review.find(params[:id])}
+  end
+
+  def new
+    render json: Superpower.find(params[:superpower_id])
+    render json: Review.new
+  end
+
+  def create
+    @superpower = Superpower.find(params[:superpower_id])
+    @review = Review.new(review_params)
+    @review.user = current_user
+    @review.superpower = @superpower
+
+    if @review.save
+      flash[:notice] = 'Review Added Successfully'
+      redirect_to superpower_path(@superpower)
+    else
+      flash[:alert] = @review.errors.full_messages.join(", ")
+      render :new
+    end
+  end
+end
```

Comment:
I don't see any method `review_params` here - can we get some controller api tests in here to make sure this is working? I'm fairly certain this endpoint isn't functional right now.